server/services/booking.js

```diff
@@ -133,6 +136,18 @@ const bookingServiceFactory = (eliteApi, keyworkerApi) => {
         : null
     }
 
+    const getPrisonOffenderManagerName = pom => {
+      if (pom.name) {
```

Comment:
Do we need to check to see if pom is defined?

Reply:
This is only called if there is a pom:
```
prisonOffenderManager:
          prisonOffenderManagerData.primary_pom && getPrisonOffenderManagerName(prisonOffenderManagerData.primary_pom),
        coworkingPrisonOffenderManager:
          prisonOffenderManagerData.secondary_pom &&
          getPrisonOffenderManagerName(prisonOffenderManagerData.secondary_pom)
```